### ISSUE TITLE
disallowBlockExpansion causes false positives on namescaped attributes

### ISSUE THREAD

**Issue by benedfit** [pugjs/pug-lint#62]
e.g.

```
svg
  use(xlink:href='#icon-pencil')
```

**Comment by benedfit** [comment]
User error, this is not a bug
